lib/hyperclient/entry_point.rb

```diff
@@ -112,6 +113,18 @@ def faraday_block=(value)
       @faraday_block = value
     end
 
+    # Public: Set options.
+    #
+    # value    - A Hash containing the client options. Use { async: false } to
+    # to disable the default behavior of performing requests asynchronously
+    # using futures.
+    attr_writer :options
+
+    # Public: Read options.
+    #
+    # Returns a Hash.
+    attr_reader :options
```

Comment:
I separated them to have documentation on both, but it may not be useful enough to warrant that distinction.